packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/module.js

```diff
@@ -5,11 +5,12 @@ import routing from './routing';
 import service from './service';
 
 import list from './list';
+import deletion from './list/delete';
```

Comment:
```suggestion
import delete from './list/delete';
```

Reply:
This is not possible as `delete` is an operator